mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -3156,6 +3157,10 @@ fn lp_connected_alice(ctx: MmArc, taker_order: TakerOrder, taker_match: TakerMat
                         uuid,
                         p2p_keypair: taker_order.p2p_privkey.map(SerializableSecp256k1Keypair::into_inner),
                         taker_secret,
+                        maker_p2p_pubkey: match maker_p2p_pubkey {
+                            PublicKey::Secp256k1(pubkey) => pubkey.into_inner(),
+                        },
```

Comment:
It's worth to implement `From` trait on `PublicKey::Secp256k1` I think.

Reply:
According to https://github.com/KomodoPlatform/komodo-defi-framework/pull/2046#discussion_r1457161569, I changed this to
```rust
maker_p2p_pubkey: match maker_p2p_pubkey {
    PublicKey::Secp256k1(pubkey) => pubkey.into(),
},
```

Is that ok for now?

Reply:
I meant doing `match` on single variant value seems different (considering we do this on multiple spots), it could be better to do it like `maker_p2p_pubkey: maker_p2p_pubkey.into()`.

Reply:
It should be `TryFrom/TryInto` then because we may have new variants of `mm2_p2p::PublicKey` not convertible to `Secp256k1Pubkey`. Having a single variant enum might look different in general, so I would keep the things as is for now and refactor it later when we add more variants :slightly_smiling_face: